content/learning-paths/smartphones-and-mobile/using-neon-intrinsics-to-optimize-unity-on-android/7-analyzing.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Analyzing the performance data
+weight: 8
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+If you need a refresh on the Unity Analyzer, read [Profiling Unity apps on Android](/learning-paths/smartphones-and-mobile/profiling-unity-apps-on-android) before you continue.
+
+## Pull data into the Analyzer
+
+To compare the results you will use the Analyzer to import all three datasets but we will only compare two at a time:
+
+1. Open the Profiler window
+
+1. Select _Clear_ to clear the results
+
+1. Select the folder icon in the top right to load the binary recording _plain.data_
+
+1. Open the Profile Analyzer from _Window/Analysis_ menu
+
+1. Set the Analyzer to mode Compare (in the top-left)
+
+1. Slots for two datasets will appear (both with _Pull Data_, _Load_ and _Save_ buttons)
+
+1. Select the _Pull Data_ button from the top. The data will import from the Profiler.
+
+1. Click back to the Profiler window and _Clear_ the data
+
+1. Select folder icon to load the binary data for _burst.data_
+
+1. Click back to the Analyzer and select the second _Pull Data_.
+
+You now have two datasets loaded; one from the unoptimized mode and one from the Burst mode. This is what the data looks like collected from a Samsung S8.
```

Comment:
"This is what the data looks like", but the screenshot is further down. "Below is what the data looks like" or move the screen-shot up, or remove this sentence?

Reply:
or add a screenshot, I guess that's the 4th option.

Reply:
i moved the screenshot up a bit - i think it breaks the text up better.
i'm not sure another screenshot would have added anything that the existing one doesn't have.
done